torch/_torch_docs.py

```diff
@@ -7130,6 +7130,9 @@ def merge_dicts(*dicts):
 
 Returns the total number of elements in the :attr:`input` tensor.
 
+This function can also be used with a ``torch.Size`` tensor, but has to be called on it directly and
```

Comment:
I was wrong about this and gave you some bad advice with https://github.com/pytorch/pytorch/issues/61231#issuecomment-880863537. Sorry about that @stas00.

torch.Size is defined here 

https://github.com/pytorch/pytorch/blob/968a01a94a0465c37cb6a60ca2f3c430f4c062b9/torch/csrc/Size.cpp#L49

and it defines the numel method here:

https://github.com/pytorch/pytorch/blob/968a01a94a0465c37cb6a60ca2f3c430f4c062b9/torch/csrc/Size.cpp#L141

But it can't be passed to the `torch.numel()` function:

```
t = torch.tensor((1, 2, 3))
torch.numel(t.size())
: TypeError: numel(): argument 'input' (position 1) must be Tensor, not torch.Size
```

But the numel method on a torch.Size object works:

```
t.size().numel()
: 3
```

So we'd need to find the documentation for torch.Size (if it exists) and then document this method.